src/components/Forms/MultiAmountSelector.vue

```diff
@@ -27,11 +27,13 @@
 				>
 					<kv-currency-input
 						id="custom-amount-input"
+						ref="kvCurrencyRef"
 						class="input-element"
-						:class="{'custom-input-element': true, 'error': $v.$invalid }"
+						:class="{'custom-input-element': true, 'error': $v.$invalid}"
 						v-model="customAmountModel"
 						:aria-disabled="!selected === 'custom'"
 						:disabled="!selected === 'custom'"
+						@focus.native="$event.target.select()"
```

Comment:
Is this guy doing anything?

Reply:
If you have 1 instance of this component that just sits on the page it will select the text once you click into the input (after you've given focus to something else that is).

My implementation using focus between instances in the parent completely overrides it though.

I can remove it.

Reply:
gone